tracer/src/Datadog.Trace/Telemetry/Metrics/PublicApiUsage.cs

```diff
@@ -23,7 +23,13 @@ internal enum PublicApiUsage
     [Description("name:eventtrackingsdk_trackuserloginsuccessevent_metadata")]EventTrackingSdk_TrackUserLoginSuccessEvent_Metadata,
 
     [Description("name:spancontextextractor_extract")] SpanContextExtractor_Extract,
+
+    [Description("name:spancontextextractor_extractincludingdsm")]
+    SpanContextExtractor_ExtractIncludingDsm,
```

Comment:
I'm wondering if we should put these at the end so they don't change the numbers, but need to double check if it's an issue

Reply:
meh, they're grouped with the other extractor values, I think it's fine